coalib/results/Result.py

```diff
@@ -287,7 +287,7 @@ def overlaps(self, ranges):
         """
         Determines if the result overlaps with source ranges provided.
 
-        :param ranges: A list SourceRange objects to check for overlap.
+        :param ranges: A list ``SourceRange`` objects to check for overlap.
```

Comment:
Can you change it back? The issue requested the change in the line number 72-73 only.
You need to rebase too.